### PR TITLE
fix: Make app compatible with latest auto-prefixer

### LINKED ISSUE #4119: Application not building
Application not building on local after updating branch with upstream

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: kushthedude <kushthedude@gmail.com>

Fixes #4119

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5030305&bid=16574960)

File: ember-cli-build.js [hunk 1]
```diff
@@ -13,9 +13,14 @@ module.exports = function(defaults) {
       includePolyfill: true
     },
     storeConfigInMeta : true,
-    autoprefixer      : {
-      browsers : targets.browsers,
-      cascade  : false
+    sassOptions: {
```
Comment (niranjan94) [inline ember-cli-build.js]: Codacy Issue found: [Missing space after key 'sassOptions'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5030305)

File: ember-cli-build.js [hunk 2]
```diff
@@ -13,9 +13,14 @@ module.exports = function(defaults) {
...
+      sourceMapEmbed : true
```
Comment (niranjan94) [inline ember-cli-build.js]: Codacy Issue found: [Extra space after key 'sourceMapEmbed'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5030305)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5030305&bid=16574960)

File: ember-cli-build.js [hunk 3]
```diff
@@ -13,9 +13,14 @@ module.exports = function(defaults) {
       includePolyfill: true
     },
     storeConfigInMeta : true,
-    autoprefixer      : {
-      browsers : targets.browsers,
-      cascade  : false
+    sassOptions : {
```
Comment (niranjan94) [inline ember-cli-build.js]: Codacy Issue found: [Missing space after key 'sassOptions'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5030305)

**Comment by iamareebjamal** [comment]
How did it build in CI then? And on netlify?

**Comment by kushthedude** [comment]
Environment production

**Pull request merged**